docs/standard/data/sqlite/custom-versions.md

```diff
@@ -41,14 +38,51 @@ Install-Package SQLitePCLRaw.bundle_e_sqlcipher
 
 ---
 
-## SQLitePCLRaw providers
+## SQLitePCLRaw available providers
+
+When not relying on a bundle, you can use the available providers of SQLite with the core assembly.
+
+| Provider | Description |
+|--|--|
+| [SQLitePCLRaw.provider.dynamic](https://www.nuget.org/packages/SQLitePCLRaw.provider.dynamic) | The `dynamic` provider loads the native library instead of using <xref:System.Runtime.InteropServices.DllImportAttribute?displayProperty=nameWithType> attributes. For more information on using this provider, see [`SQLitePCLRaw` custom providers](#sqlitepclraw-custom-providers). |
```

Comment:
```suggestion
| [SQLitePCLRaw.provider.dynamic](https://www.nuget.org/packages/SQLitePCLRaw.provider.dynamic) | The `dynamic` provider loads the native library instead of using <xref:System.Runtime.InteropServices.DllImportAttribute?displayProperty=nameWithType> attributes. For more information on using this provider, see [use the dynamic provider](#use-the-dynamic-provider). |
```